proposals/Proposal.md

```diff
@@ -0,0 +1,113 @@
+#﻿Improving reproducibility in science by adding provenance tracking to the EcoData Retriever.
+
+
+#Personal Details:-
+Name:- Prabh Simran Singh Baweja
+3rd Year, B.Tech in Computer Science and Engineering, IIIT Hyderabad, India.
+Email Id:- prabhsimransingh.baweja@gmail.com
+Website:- web.iiit.ac.in/~prabhsimransingh.baweja
+Phone Number:- +91-8790616675.
+
+
+#Abstract:
+The EcoData Retriever is a Python based tool for automatically downloading, cleaning up, and restructuring ecological data. Most of the steps are either related to data are either done manually, or using one-off scripts. Due to this, the process is not reproducible. The EcoData Retriever does most of this work, but is not able to keep track of the work that has been completed and therefore fails to support full reproducible workflows.

```

Comment:
"Most of the steps are either related to data" -> "Most of the steps related to data acquisition and manipulation"